### ISSUE TITLE
[bug] v2.2.5 malformed namespaces

### ISSUE THREAD

**Issue by Raerten** [centrifugal/centrifugo#319]
**Describe the bug**

error 'malformed namespaces'

```
PS C:\centrifugo> .\centrifugo.exe genconfig
PS C:\centrifugo> .\centrifugo.exe
{"level":"error","error":"unexpected end of JSON input","time":"2019-11-29T11:19:58+03:00","message":"malformed namespaces"}
PS C:\centrifugo> cat .\config.json
{
  "secret": "f3864e6f-d1cb-4892-b621-c109782270f3",
  "admin_password": "c795b00f-fe4d-45dd-9987-daf1459173cd",
  "admin_secret": "5b951424-2a04-410b-9581-cafe41170f0b",
  "api_key": "dec17255-dff3-434d-8c9e-d5b5120de609"
}
```

**Versions**
Centrifugo v2.2.5 (Go version: go1.13.4)
windows and docker image

**Comment by FZambia** [comment]
@Raerten thanks a lot! Will fix today in the evening.

As temporary workaround add `"namespaces": []` to configuration file.

**Comment by drezo** [comment]
@FZambia thx!

**Comment by FZambia** [comment]
Released v2.2.6, hope there won't be any cases I missed related to configuration. Sorry for this, guys.
